modules/exploits/windows/scada/realwin_seh_buffer_overflow.rb

```diff
@@ -0,0 +1,72 @@
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::Tcp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'DATAC RealWin SCADA Server Buffer Overflow',
+      'Description'    => %q{
+          This module exploits a stack buffer overflow in DATAC Control
+        International RealWin SCADA Server 2.0 (Build 6.0.10.37).
+        By sending a specially crafted packet to port 912,
+        an attacker may be able to execute arbitrary code.
```

Comment:
Presumably this module does not bypass DEP/ASLR, which should probably be mentioned here.